### PR TITLE
Microbuild sign

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

Move signing to use MicroBuild

### Issues Resolved ### 

None

### API Changes ###

 None

### Platforms Affected ### 

- All

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Check that the signed packages are the release one makse sure this works with a tag 

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ resources:
       type: github
       name: xamarin/yaml-templates
       endpoint: xamarin
-      ref: refs/heads/main # still defaults to master even though main is the main branch
+      ref: refs/heads/dev/signing-folders # still defaults to master even though main is the main branch
```
Comment (rmarinho) [inline azure-pipelines.yml:29]: @mattleibow  we can merge this on main ?

File: build/steps/build-sign.yml [hunk 2]
```diff
@@ -11,15 +11,10 @@ parameters:
 steps:
   - checkout: none
     clean: true
-
```
Comment (rmarinho) [inline build/steps/build-sign.yml:14]: this file can go.

File: azure-pipelines.yml (hunk 1 above)
Comment (mattleibow) [reply azure-pipelines.yml:29]: merged!

File: build/SignList.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  <ItemGroup>
+    <FirstParty Include="Xamarin.Forms.*.dll" />
+    <FirstParty Include="FormsViewGroup.dll" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ThirdParty Include="GMap.NET.Core.dll" />
+    <ThirdParty Include="GMap.NET.GTK.dll" />
+    <ThirdParty Include="webkit-sharp.dll" />
+  </ItemGroup>
```
Comment (mattleibow) [inline build/SignList.xml:12]: Mono.Cecil needs to be added here. 

https://github.com/xamarin/AndroidX/pull/193

```suggestion
    <ThirdParty Include="Mono.Cecil.dll" />
    <ThirdParty Include="Mono.Cecil.*.dll" />
  </ItemGroup>
```

File: azure-pipelines.yml (hunk 1 above)
Comment (rmarinho) [reply azure-pipelines.yml]: thanks!

File: build/SignList.xml (hunk 3 above)
Comment (rmarinho) [reply build/SignList.xml:14]: Added!

**Review by mattleibow — APPROVED** [review]

**Pull request merged**